docs/indicator-configuration.md

```diff
@@ -290,6 +292,17 @@ graph_limits:
     maximum: 2000
 ```
 
+### graph_series_breaks
+
+**_Optional_**: This setting can be used to add "series break" annotations (ie, gaps in the years) to the graph. Like `graph_titles` it can include multiple items, and each can be limited to particular units or series. Each item can have the following settings:
+
+    * `series`: If specified, the series break will only display when the user is looking at this series.
+    * `unit`: If specified, the series break will only display when the user is looking at this unit of measurement.
+    * `label`: The text to display on the annotation. Defaults to "2030 target".
+    * `value`: The value at which to draw the line. This number should be between 0 (the left side of the chart) and the number of years minus 1 (the right side of the chart).
+
+Note that this setting relies on the Chart.js annotation plugin. If you would like to tweak any other settings for more site-wide control, you can override the `_includes/components/annotation_presets.js` file.
```

Comment:
Is this supposed to be `_includes/components/charts/annotation_presets.js`?

Reply:
@LucyGwilliamAdmin Ah, that's right, I'll update those docs. Did you already try that or no?

Reply:
Do you mean where I'm trying to overwrite? Yeah, I've put it in the charts folder

Reply:
Got it - OK I'll investigate.